app/models/order.rb

```diff
@@ -0,0 +1,16 @@
+class Order < ApplicationRecord
+  belongs_to :client
+  belongs_to :staff
+  has_many :line_items
+  validates :total, presence: true
+  validates :subtotal, presence: true
+  validates :tax, presence: true
+
+  # TODO: write code to grab line_item_total for each line_item to get subtotal for order
+  def calculate_subtotal(line_items)
+  end
+
+  def calculate_tax
+    calculate_subtotal * 0.00575
```

Comment:
I'm worried about hardcoding the tax without making a method somewhere down the road (not necessarily now) to change it. I think at some point we're going to need to make calculate tax take an argument and allow the admin to change that argument. Just a note